### DISCUSSION TITLE
Welcome to frontend Discussions!

### DISCUSSION THREAD

**Discussion by NtemKenyor** [KwickerHub/frontend#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by codewithmide** [comment]
Hello devs,

I'm Olumide Micheal a frontend web developer. 

Regarding this project, I would like to know if there is a prototype that has to be followed by the contributors. Because, there are lots of fixes that has to be done in order to optimise the website and improve responsiveness on different devices.

I look forward to hearing your response.

Thank you!

**Comment by NtemKenyor** [comment]
well at the moment, 70 to 80 percent of the items we have developed already did not come from a UI designer or a Prototype maker. Eventually we will surely design and redesign tools and websites.

I also appreciate you for your interest in this program. It's not very easy to correct already existing work and manage the community all together . However, About the bugs, Please give detailed explanation of the problem idea and we will move into action. You can also raise issues here on github. have a nice day

**Comment by codewithmide** [comment]
Thank you for your response. I'll speak to a friend who is a UI/UX designer if she can come up with a prototype, color pallets, doodles, icons etc.

I'm so much interested in contributing to this project!🚀

**Reply by NtemKenyor** [reply]
Please do the entire team is behind you. Thank you @codewithmide

**Comment by JasmineDavidson94** [comment]
Hello, I am reaching out as a graduate student looking to contribute to this open source project. I would like to complete a couple pull requests for the good first issues listed. I will complete the requests and merge within 24hrs. Thank you-Jasmine

**Reply by JasmineDavidson94** [reply]
Hello, I recently made changes to a pull request and also created 2 new pull request. Can you please review and merge these requests?

**Comment by Ionfinisher** [comment]
Hello everyonne, I'm having issues cloning the project. Can someone please help me?

**Comment by Bluestar391** [comment]
Hey! I am Emma i am sorry but can anyone of you explain this app for me i want to know how to use it....
I am waiting for and thanks 💗
